### PR TITLE
completed html-forms

### PR DESCRIPTION
**Pull request by tinachia**

### CODE REVIEW

**Comment by tinachia** [comment]
https://watch.screencastify.com/v/EG0ZxkrvIljoPobSOUSE

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        <option value="Select an option">Select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+    </form>
+    <form>
+      <div>Favorite Language (choose one)
+        <br />
+        <p>
```
Comment (Zhiwutian) [inline html-forms/index.html:26]: this should not be in a p tag

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          <input type="radio" name="type" value="HTML" checked="checked" /> HTML
```
Comment (Zhiwutian) [inline html-forms/index.html:27]: each of these inputs needs a label with matching for and id attributes

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <h1>Coding Form!</h1>
+    <p>
+    <form>
+      <div>email:
+        <div><input type="text" name="email" size="15" maxlength="30" /></div>
+      </div>
+    </form>
+    <form>
+      <div>Why is coding cool?</div>
+      <select name="Select an option">
```
Comment (Zhiwutian) [inline html-forms/index.html:16]: this input needs a label with matching for and id attributes

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>
+    <form>
+      <div>email:
+        <div><input type="text" name="email" size="15" maxlength="30" /></div>
```
Comment (Zhiwutian) [inline html-forms/index.html:11]: this input needs a label with matching for and id attributes

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+          CSS
+          <input type="radio" name="type" value="JavaScript" />
+          JavaScript
+        </p>
+      </div>
+    </form>
+    <form>
+      <div>Would you like to sign up for more information?
+        <br />
+        <p><input type="checkbox" name="service" value="Sign me up!" checked="checked" /> Sign me up!</p>
```
Comment (Zhiwutian) [inline html-forms/index.html:38]: this input needs a label with a matching for and id attributes

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>
+    <form>
```
Comment (Zhiwutian) [inline html-forms/index.html:9]: there should be only one `form` tag on the whole form

File: html-forms/index.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>
```
Comment (Zhiwutian) [inline html-forms/index.html:8]: remove this p tag

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
some updates are needed, please reach out with questions if needed

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**